docs/breakingchanges-rc2.md

```diff
@@ -0,0 +1,15 @@
+# Breaking Changes for C#/WinRT in RC2
```

Comment:
Did you want to mention the 'add' event accessor changing return type as well?  (from EventRegistrationToken to void)